api/clamav_scanner/common.py

```diff
@@ -5,10 +5,8 @@
 
 AV_DEFINITION_S3_BUCKET = os.getenv("AV_DEFINITION_S3_BUCKET")
 AV_DEFINITION_S3_PREFIX = os.getenv("AV_DEFINITION_S3_PREFIX", "clamav_defs")
-AV_DEFINITION_PATH = os.getenv(
-    "AV_DEFINITION_PATH",
-    "/tmp/clamav",  # nosec - [B108:hardcoded_tmp_directory] Lambdas can only write to the /tmp folder
-)
+AV_DEFINITION_PATH = os.getenv("AV_DEFINITION_PATH", "/clamav")
```

Comment:
I think you just need the same `# nosec` comment on these to get Bandit to calm down 😅

Reply:
it was the line below that had `/tmp`. Fixed!